Rendera/src/main.cpp

```diff
@@ -4,93 +4,143 @@
 #include "camera.hpp"
 #include "sphere.hpp"
 #include "intersection.hpp"
+#include "scene.hpp"
 #include <fstream>
 #include <cmath>
 
-color lighting(Material mat, ray cam_ray, ray normal, vec3 point, vec3 light_src, color intensity){
+// color lighting(Material mat, ray cam_ray, ray normal, vec3 point, vec3 light_src, color intensity){
```

Comment:
Why did you comment all of this instead of deleting it? Since we are using Git, there is no fear of losing work because all the history is stored by Git itself.